### PR TITLE
chat: add dark mode

### PR DESCRIPTION
**Pull request by matildepark**
Adds custom classes and declarations for dark mode color scheme.
Uses [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query to check preference.

Note that prefers-color-scheme is set to 'dark' when:
- The operating system (mobile or desktop) is in 'dark mode' or
- Your web browser is set to 'dark mode'

Screen Shot 2019-12-27 at 12 10 30 PM
Screen Shot 2019-12-27 at 12 12 35 PM

### CODE REVIEW

File: pkg/interface/chat/src/css/custom.css [hunk 1]
```diff
@@ -141,4 +141,46 @@ h2 {
   .h-100-minus-48-xl {
     height: calc(100% - 48px);
   }
+}
+
+/* dark */
+
```
Comment (urcades) [inline pkg/interface/chat/src/css/custom.css]: Another thing to consider for a dark theme are images set at a lower opacity, perhaps around 60%.

This is another eyesight saver — nothing punches a hole in a dark theme like a bright ass image coming into view.

Comment (urcades) [reply pkg/interface/chat/src/css/custom.css]: Everything else seems solid tho, excited to see this live.

**Comment by jtobin** [comment]
ping @loganallenc

**Comment by jtobin** [comment]
Gonna merge this one as I reckon there's not much controversy around it.

**Pull request merged**